samples/excel/82-document/get-file-in-slices-async.yaml

```diff
@@ -0,0 +1,201 @@
+id: excel-82-document-get-file-in-slices-async-
+name: Get file in slices and recombine
```

Comment:
Suggest that you change the name to:  Get file

Reply:
Done.